packages/teams-js/src/private/externalAppCardActionsForCEC.ts

```diff
@@ -0,0 +1,85 @@
+import { sendMessageToParentAsync } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { ApiName, ApiVersionNumber, getApiVersionTag } from '../internal/telemetry';
+import { validateId } from '../internal/utils';
+import { errorNotSupportedOnPlatform, FrameContexts } from '../public/constants';
+import { runtime } from '../public/runtime';
+import { externalAppCardActions } from './externalAppCardActions';
+
+/**
+ * Updated to constants file: v2 APIs telemetry file: All of APIs in this capability file should send out API version v2 ONLY
+ */
+const externalAppCardActionsTelemetryVersionNumber: ApiVersionNumber = ApiVersionNumber.V_2;
+
+export namespace externalAppCardActionsForCEC {
+  export function processActionOpenUrl(
+    appId: string,
+    conversationId: string,
+    url: URL,
+  ): Promise<externalAppCardActions.ActionOpenUrlType> {
+    ensureInitialized(runtime, FrameContexts.content);
+
+    if (!isSupported()) {
+      throw errorNotSupportedOnPlatform;
+    }
+    validateId(appId, new Error('App id is not valid.'));
+    return sendMessageToParentAsync<
+      [externalAppCardActions.ActionOpenUrlError, externalAppCardActions.ActionOpenUrlType]
+    >(
+      getApiVersionTag(
+        externalAppCardActionsTelemetryVersionNumber,
+        ApiName.ExternalAppCardActions_ProcessActionOpenUrl,
+      ),
+      'externalAppCardActions.cec.processActionOpenUrl',
```

Comment:
It should be 'externalAppCardActionsForCec,processActionOpenUrl`

Reply:
Updated to `externalAppCardActionsForCEC.processActionOpenUrl` to align with the authCEC. Let me know if you prefer the `Cec` one. I'm ok with eigher way.